dvc/main.py

```diff
@@ -75,6 +86,9 @@ def main(argv=None):
         # so won't be reused by any other subsequent run anyway.
         clean_repos()
 
+    if ret != 0:
+        logger.info(footer)
```

Comment:
We used to print that message to stderr, and now we are going to print it to stdout. It would be great if we could keep it in stderr, if it requires a reasonable amount of effort. Otherwise, we could keep it in stdout as is for now.

Also, just wanted to point this out for the record, this helper message won't be shown to anyone using dvc through API now, and I think that is acceptable, as its primary purpose is for the CLI users. We might consider bringing these back in the future if we publically release such a low-level API.